examples/08_experimental/plot_3d_map_to_surface_projection.py

```diff
@@ -0,0 +1,341 @@
+"""
+Making a surface plot of a 3D statistical map
+=============================================
+
+.. warning::
+
+    This is an adaption of
+    :ref:`sphx_glr_auto_examples_01_plotting_plot_3d_map_to_surface_projection.py`
+    to use make it work with the new experimental surface API.
+
+In this example, we will project a 3D statistical map onto a cortical mesh
+using :func:`~nilearn.surface.vol_to_surf`,
+display a surface plot of the projected map
+using :func:`~nilearn.plotting.plot_surf_stat_map`
+with different plotting engines,
+and add contours of regions of interest using
+:func:`~nilearn.plotting.plot_surf_contours`.
+
+"""
+
+# %%
+# Get a statistical map
+# ---------------------
+
+from nilearn import datasets
+
+stat_img = datasets.load_sample_motor_activation_image()
+
+
+# %%
+# Get a cortical mesh
+# -------------------
+
+from nilearn.experimental.surface import load_fsaverage
+
+fsaverage_meshes = load_fsaverage()
+fsaverage = datasets.fetch_surf_fsaverage()
+
+# %%
+# Use :term:`mesh` curvature to display useful anatomical information
+# on inflated meshes
+#
+# Here, we load the curvature map of the hemisphere under study,
+# and define a surface map whose value for a given :term:`vertex`
+# is 1 if the curvature is positive,
+# -1 if the curvature is negative.
+
+import numpy as np
+
+from nilearn import surface
+
+curv_right = surface.load_surf_data(fsaverage.curv_right)
+curv_right_sign = np.sign(curv_right)
+
+# %%
+# Sample the 3D data around each node of the mesh
+# -----------------------------------------------
+from nilearn.experimental.surface import SurfaceImage
+
+img = SurfaceImage(
+    mesh=fsaverage_meshes["pial"],
+    data={
+        "left": surface.vol_to_surf(
+            stat_img, fsaverage_meshes["pial"]["left"]
+        ),
+        "right": surface.vol_to_surf(
+            stat_img, fsaverage_meshes["pial"]["right"]
+        ),
+    },
+)
```

Comment:
projecting to surface will probably be a fairly classic use case that it would probably justify having a methods for it.